include/okapi/api/chassis/controller/chassisController.hpp

```diff
@@ -57,6 +57,7 @@ class ChassisController {
 
   /**
    * Turns the robot clockwise in place (using closed-loop control).
+   * (turns the shortest angle for OdomChassisControllers)
```

Comment:
This isn't applicable to all chassis controllers, so it shouldn't be documented here. You can put it [here](https://github.com/OkapiLib/OkapiLib/blob/master/include/okapi/api/chassis/controller/defaultOdomChassisController.hpp) though. Also please rephrase it to be clearer, e.g. "Turns in the direction of the smallest angle"